### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by ClementNerma** [iron/iron#635]
Hi!

I've recently came across this repository and found it did not seem to be actively maintained (only ~100 commits in 3 years and a half).

Is this project still actively maintained?

Otherwise, wouldn't it be a good idea to indicate the project is looking for maintainers?

It'd be unfortunate for such a great project to be let down now it's that advanced.

Thanks in advance for your answers :)

**Comment by phlmn** [comment]
The honest answer is: **barely...**

I tried to reanimate the project (https://github.com/iron/iron/issues/578) but don't find much time. Also, my motivation faded since I don't have enough permissions to do some things, and I failed to reach someone who has permissions to do so (for example update GitHub pages settings to redo the landing page etc.).

I support the idea of indicating the need for maintainers, but without granting more permissions or have someone with proper permissions available, new maintainers it won't get very far.

I think iron needs a maintainer (or a few) that identify with the project and decide in which direction it should evolve to differentiate itself from newer frameworks like Actix Web, Warp and tide.

I'm happy to support getting this project back on track, but at the moment I don't have the time or motivation to be the one leading it.
